styles.css

```diff
@@ -1,20 +1,194 @@
 * {
+    -webkit-box-sizing: border-box;
     box-sizing: border-box;
+    font-family: 'Catamaran', sans-serif;
 }
 
 body {
     font-size: 62.5%;
+    background-image: url("./img/34216.jpg");
+    background-attachment: fixed;
+}
+
+h1 {
+    font-size: 3rem;
+    margin: 0;
+    text-align: center;
+}
+
+p {
+    font-size: 2rem;
+    text-align: center;
 }
 
 img {
-    width: 200px;
+    height: 90%;
+}
+
+.container {
+    background-color: rgba(255, 255, 255, 0.6);
+    padding: 20px;
+}
+
+.guitarsContainer {
+    border: 1px gray solid;
+    background-color: rgba(240, 240, 255, 0.8);
+    width: 90%;
+    margin: 0 auto;
+    font-size: 1.6rem;
 }
 
 .row {
-    overflow:auto;
+    height: 24vh;
+}
+
+.header {
+    font-weight: bold;
+    height: 14vh;
 }
 
 .item-detail {
-    float:left;
-    width: 200px;
+    float: left;
+    width: 9%;
+    height: 100%;
+    padding: 10px;
+    text-align: center;
+    border: 1px gray solid;
+}
+
+.column1 {
+    width: 4%;
+}
+
+.column2 {
+    width: 25%;
+}
+
+.item .column2{
+    background-color: rgba(240, 240, 255, 0.8);
+}
+
+.column3 {
+    width: 16%;
+}
+
+.column6 {
+    width: 12%;
+}
+
+.column8 {
+    width: 16%;
+}
+
+@media screen and (max-aspect-ratio: 4/3) {
+    img {
+        width:70%;
+        height: auto;
+    }
+}
+
+@media screen and (max-height: 600px) {
+    .item {
+        height: 35vh;
+    }
+
+    .guitarsContainer {
+        font-size: 1rem;
+    }
+
+
+}
+
+@media screen and (max-height: 750px) {
+    .item {
+        height: 30vh;
+    }
+
+    .guitarsContainer {
+        font-size: 1.4rem;
+    }
+
+    h1 {
+        font-size: 2rem;
+    }
+
+    p {
+        font-size: 1.4rem;
+    }
+}
+
+@media screen and (max-width: 1250px) {
```

Comment:
Screen width between 1000px and 900px wraps the text "Date Acquired" onto two lines and it overflows the border

Reply:
When the screen width is less than 900px the width of guitarsCollection is more than the total width of the columns